app/shared/services/AuthService.js

```diff
@@ -187,7 +187,8 @@ class AuthService {
 						if (scope.validateState(response.data, workgroupId, year, ignoreFallBackUrl)) {
 							// Log the user to Google Analytics (only in production mode)
 							if(ipaRunningMode === 'production') {
-								ga('set', 'userId', response.data.userTrackingId);
+								window.ipa_user_tracking_id = String(response.data.userTrackingId);
```

Comment:
This `window.ipa_user_tracking_id` line shouldn't be here. `develop` was not updated with `master`. I've corrected it. Be sure to get the current AuthService.js.